src/test/compile-fail/const-block-non-item-statement.rs

```diff
@@ -10,10 +10,17 @@
 
 enum Foo {
     Bar = { let x = 1; 3 }
+<<<<<<< HEAD
     //~^ ERROR let bindings in constants are unstable
     //~| ERROR statements in constants are unstable
     //~| ERROR let bindings in constants are unstable
     //~| ERROR statements in constants are unstable
+||||||| merged common ancestors
```

Comment:
rebase leftovers

Reply:
You'd think Git wouldn't let me do this... grr....